### ISSUE TITLE
Set use_flat_hand=True as default to make retargeting easier

### ISSUE THREAD

**Issue by yl-1993** [open-mmlab/mmhuman3d#23]
In original SMPL-X, [`use_flat_hand`](https://github.com/vchoutas/smplx/blob/master/smplx/body_models.py#L516) is set to `False` by default, however, the T-pose of avatar usually has flat hand.

To make retargeting easier, we may add `use_flat_hand: bool = True` in [utils/smpl.py](https://github.com/open-mmlab/mmhuman3d/blob/main/mmhuman3d/models/utils/smpl.py#L209) to make training, inference and visualization work consistently.

**Comment by JasonBoy1** [comment]
@caizhongang This should be changed in body_model

**Comment by caizhongang** [comment]
Currently, the discrepancy is caused by mismatch of body model configurations in model training and visualization. While the default `use_flat_hand=True` is added in utils/smpl.py, the [visualize_smpl](https://github.com/open-mmlab/mmhuman3d/blob/main/mmhuman3d/core/visualization/visualize_smpl.py#L171) may also consider some improvements:
1. the user *must* input either body model or body model config. A possible way is
```
if isinstance(body_model, dict):
    ...
elif isinstance(body_model, torch.nn.Module):
    ...
else:
   raise TypeError(f'body_model should be either dict or '
                            f'torch.nn.Module, but got {type(body_model)}')
```
2. throw error if 1 is not met, but do *not* use a [hard-coded config](https://github.com/open-mmlab/mmhuman3d/blob/main/mmhuman3d/core/visualization/visualize_smpl.py#L178-L185)

**Comment by caizhongang** [comment]
Addressed in #32
